run_iwyu_tests.py

```diff
@@ -206,14 +206,8 @@ def RunOneTest(self, filename):
     files_to_check = [PosixPath(f) for f in files_to_check]
 
     iwyu_flags = self._iwyu_flags_map.get(filename, None)
-    if iwyu_flags:
-      logging.info('%s: Using iwyu flags %s', filename, str(iwyu_flags))
-
     clang_flags = self._clang_flags_map.get(filename, [])
     clang_flags.extend(self._include_map.get(filename, []))
-    if clang_flags:
-      logging.info('%s: Using clang flags %s', filename, str(clang_flags))
-
```

Comment:
I see it turned out not to be particularly useful. `-Xiwyu` is sufficient to tell if flag is IWYU flag or Clang flag.